packages/components/src/TCRUserDashboard/DashboardNewsroomStripeConnect.tsx

```diff
@@ -191,10 +194,12 @@ export class DashboardNewsroomStripeConnectComponent extends React.Component<
   }
 
   private renderStripeConnectButton(linkText?: string): JSX.Element {
-    // @TODO/toby Our stripe client ID (which should be different on production) should be in some config, but we don't have any config for components package, would have to pass in from dapp via prop or add to civil context.
-    const url =
-      "https://connect.stripe.com/oauth/authorize?response_type=code&client_id=ca_BzqgUsw7tnnCpVaoQ157mrxtuCdN7h2q&scope=read_write&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fdashboard%2Fnewsrooms";
-    return <p>{linkText ? <a href={url}>{linkText}</a> : <StripeConnectButtonLink href={url} />}</p>;
+    // @TODO/toby Remove `stripe-admin` feature flag query string param when launched
```

Comment:
alternatively you could just pop the feature flag into the .env file (`REACT_APP_FEATURE_FLAGS`) - they are overridden at build time for staging and prod anyways but allows it to work locally

Reply:
Oh sweet yeah I saw those and was going to ask to make sure they get overridden on staging/prod.

Thing is though if we do want to, say, test on prod with the feature flag not enabled by default yet, we still need to pass feature flag in to redirect url we send to stripe in order to test it so that this component is loaded when you get back.